tests/http_client_test.py

```diff
@@ -16,7 +16,7 @@ def mock_client_session():
         yield _mock
 
 
-@pytest.fixture(params=RunMode)
+@pytest.fixture(params=[RunMode.THREAD])
```

Comment:
Could be possible to open an issue in order to re-enable those tests.

The main reason for asking this is that `aiobravado` could use the new version of bravado-asyncio and it will fail as the signature of the `aiobravado.http_future.HttpFuture` is not updated.
At least having an opened issue could warn users to not use the latest bravado-asyncio until aiobravado is not updated

Reply:
I think the main question is whether we want to continue supporting aiobravado. If so then we need to figure out a way to untangle the dependencies.

Honestly people will notice that it won't work whether we create an issue or not. I'm looking into it today; I'll see how I want to proceed and then do that or ticket it.